distribution/docs/src/main/resources/_contents/_quickstart/quickstart-configuring-contents.adoc

```diff
@@ -0,0 +1,15 @@
+
+=== Configuring (Quick Install)
+
+Set the configurations needed to run ${branding}
+
+.. Go to ${secure_url}/admin.
+.. Enter the default username of `admin` and the password of `admin`.
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure guest claims attributes or use defaults.
+... Select *Standard Installation*.
+... On the System Configuration page, configure any port or protocal changes desired and add any keystores/truststores needed.
```

Comment:
Typo: `protocol`. Not sure the keystore configuration should be mentioned in the quickstart guide, but if we decide to keep it, we should have a reference to the security section like above.